OpenSSL/SSL.py

```diff
@@ -318,6 +319,56 @@ def wrapper(ssl, out, outlen, in_, inlen, arg):
         )
 
 
+class _AlpnSelectHelper(_CallbackExceptionHelper):
```

Comment:
(Technically it's an initialism) :runs:

On Mon, Apr 13, 2015 at 5:14 PM, Jean-Paul Calderone <
notifications@github.com> wrote:

> In OpenSSL/SSL.py
> <https://github.com/pyca/pyopenssl/pull/120#discussion_r28279904>:
>
> > @@ -318,6 +319,56 @@ def wrapper(ssl, out, outlen, in_, inlen, arg):
> >          )
> >
> >
> > +class _AlpnSelectHelper(_CallbackExceptionHelper):
>
> (As opposed to Alpn)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pyca/pyopenssl/pull/120/files#r28279904>.
>



-- 
"I disapprove of what you say, but I will defend to the death your right to
say it." -- Evelyn Beatrice Hall (summarizing Voltaire)
"The people's good is the highest law." -- Cicero
GPG Key fingerprint: 125F 5C67 DFE9 4084